tests/test_grid3d/test_grid.py

```diff
@@ -391,23 +337,16 @@ def test_geometrics_reek():
     assert geom["xmin"] == pytest.approx(456620, abs=1), "Xmin cell center"
 
 
-def test_activate_all_cells(tmp_path):
-    """Make the grid active for all cells."""
-    grid = Grid(EMEGFILE)
-    logger.info("Number of active cells %s before", grid.nactive)
-    grid.activate_all()
-    logger.info("Number of active cells %s after", grid.nactive)
-
-    assert grid.nactive == grid.ntotal
-    grid.to_file(tmp_path / "emerald_all_active.roff")
+def test_activate_all_cells(emerald_grid):
+    emerald_grid.activate_all()
+    assert emerald_grid.nactive == emerald_grid.ntotal
 
 
-def test_get_adjacent_cells(tmp_path):
+def test_get_adjacent_cells(tmp_path, emerald_grid):
```

Comment:
Are we missing an `assert` in this one?

Reply:
We are indeed, added some. Although, I am not sure how to test the implementation with such a large grid. At a later point I will have to go back and add a unit test case. We could add a snapshot, but it would be very large.